app/controllers/OtherInformationController.scala

```diff
@@ -63,12 +67,23 @@ class OtherInformationController @Inject()(
     }
 
   private def onSubmit(page: QuestionPage[String], ern: String, arc: String, action: Call, mode: Mode): Action[AnyContent] =
-    authorisedDataRequestAsync(ern, arc) { implicit request =>
-      formProvider(page).bindFromRequest().fold(
-        formWithErrors =>
-          Future.successful(BadRequest(view(page, formWithErrors, action))),
-        value =>
-          saveAndRedirect(page, value, mode)
-      )
+    authorisedDataRequestAsync(ern, arc) { implicit request: DataRequest[_] =>
+      Try {
+        val trimmedFormValues: Map[String, Seq[String]] = request.body.asInstanceOf[AnyContentAsFormUrlEncoded].data.map {
+          case (k, v) => (k, v.map(_.trim))
+        }
+
+        formProvider(page).bindFromRequest(trimmedFormValues).fold(
+          formWithErrors =>
+            Future.successful(BadRequest(view(page, formWithErrors, action))),
+          value =>
+            saveAndRedirect(page, value, mode)
+        )
+      } match {
+        case Failure(exception) =>
+          logger.warn(exception.getMessage)
+          Future.successful(BadRequest(view(page, formProvider(page).withError(FormError("more-information", s"$page.error.required")), action)))
+        case Success(value) => value
+      }
```

Comment:
It would be worth taking this into a `BaseController` as it will need to be re-used on another page for the ItemShortageOrExcess controller. I don't know how you can get around the `asInstanceOf` sadly - not without refactoring the `authorised` helpers to allow you to take in a different parser than `AnyContent` which is probably more painful that just have the `try` `catch`.